modules/web-console/frontend/app/modules/agent/types/ClusterSecrets.js

```diff
@@ -35,24 +32,13 @@ export class ClusterSecrets {
     }
 
     resetCredentials() {
-        this.resetSessionToken();
-
         this.password = null;
     }
 
-    resetSessionToken() {
-        this.sessionToken = null;
-    }
-
     /**
-     * @return {{sessionToken: String}|{'user': String, 'password': String}}
+     * @return {{user: string, password: string}}
```

Comment:
Do we need password at all?
For me we need flag "authenticated: boolean" instead.

Reply:
We still need user, password for case when session expired